copycat/api/src/test/java/org/apache/kafka/copycat/data/SchemaBuilderTest.java

```diff
@@ -188,6 +190,21 @@ public void testBytesBuilderInvalidDefault() {
     }
 
 
+    @Test
+    public void testParameters() {
+        Map<String, String> expectedParameters = new HashMap<>();
+        expectedParameters.put("foo", "val");
+        expectedParameters.put("bar", "baz");
+
+        Schema schema = SchemaBuilder.string().parameter("foo", "val").parameter("bar", "baz").build();
+        assertTypeAndDefault(schema, Schema.Type.STRING, false, null);
+        assertMetadata(schema, null, null, null, expectedParameters);
+
+        schema = SchemaBuilder.string().parameters(expectedParameters).build();
+        assertTypeAndDefault(schema, Schema.Type.STRING, false, null);
+        assertMetadata(schema, null, null, null, expectedParameters);
+    }
+
```

Comment:
Appropriately concerned. I started writing a test before I realized I couldn't get a non-String value in there :)